contrib/babelfishpg_common/sql/varbinary.sql

```diff
@@ -159,7 +159,7 @@ AS 'babelfishpg_common', 'float4varbinary'
 LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
 
 CREATE CAST (REAL AS sys.BBF_VARBINARY)
-WITH FUNCTION sys.float4varbinary (REAL, integer, boolean) AS ASSIGNMENT;
+WITH FUNCTION sys.float4varbinary (REAL, integer, boolean) AS IMPLICIT;
```

Comment:
why we only need this change for INT and REAL ? what about other types such as SMALLINT, BIGINT, TINYINT etc ?

Reply:
We should add tests for other types as well if not already there.